src/components/AdjustVaultInstructions.tsx

```diff
@@ -0,0 +1,18 @@
+export const AdjustVaultInstructions = () => (
+  <div className="bg-white shadow-[0_22px_34px_rgba(116,116,116,0.25)] rounded-[10px] px-7 py-4">
+    <h3 className="text-[#484848] font-bold font-serif">Instructions</h3>
```

Comment:
the only other use of `484848` in the app palette is a drop shadow. Is there another gray we've used for this text case?

Reply:
Good point. It has a variable already `mineShaft`, but it's actually the default text color, so just removed this.